common/util/file.py

```diff
@@ -24,6 +24,7 @@ def determine_syntax_files():
 def get_syntax_for_file(filename):
     extension = get_file_extension(filename)
     syntaxes = syntax_file_map.get(filename, None) or syntax_file_map.get(extension, None)
+    # watch out if this one has changed https://github.com/sublimehq/Packages/tree/master/Text
```

Comment:
This is unnecessary - if the package changes to the new `sublime-syntax`, we will know it sooner by a new bug than by running into this comment.  If you feel we need a reminder, let's open an issue, indicating that we should review all the official syntax packages in a month or two.